daemons/fenced/fenced_commands.c

```diff
@@ -2411,15 +2411,8 @@ send_async_reply(async_command_t *cmd, const pcmk__action_result_t *result,
 
     if (bcast) {
         send_cluster_message(NULL, crm_msg_stonith_ng, reply, FALSE);
-
-    } else if (cmd->origin) {
-        crm_trace("Directed reply to %s", cmd->origin);
-        send_cluster_message(crm_get_peer(0, cmd->origin), crm_msg_stonith_ng, reply, FALSE);
-
     } else {
-        crm_trace("Directed local %ssync reply to %s",
-                  (cmd->options & st_opt_sync_call) ? "" : "a-", cmd->client_name);
-        do_local_reply(reply, cmd->client, cmd->options & st_opt_sync_call, FALSE);
+        stonith_send_reply(reply, cmd->options, cmd->origin, cmd->client);
```

Comment:
loosing the trace (apart from what send_cluster_message does) here for everything but broadcast.

Reply:
Yep, I considered the broadcast situation to be the only interesting one, though stonith_send_reply() -> do_local_reply() will trace for the IPC case. Someone can trace the low-level cluster functions if they want that level of detail for the directed replies.